userprofile/models.py

```diff
@@ -190,6 +190,7 @@ class HealthCheckUserProfile(
     tbconnect_group_arm = models.CharField(
         max_length=22, choices=GROUP_ARM_CHOICES, null=True, blank=True
     )
+    research_consent = models.BooleanField(default=False)
```

Comment:
I think both of these should default to None. so that:
None: We don't know yet
False: they've declined
True: they've consented

Reply:
Will change it, i'm thinking of closing this PR and create a new one